app/javascript/src/core_dependencies.js

```diff
@@ -0,0 +1,11 @@
+// This is to put our external/yarn imports in one spot.
+
+// Require turbolinks to not constantly do full page refreshes.
+import Turbolinks from 'turbolinks';
+Turbolinks.start();
+
+// Bootstrap 4 is our frontend
+import 'bootstrap';
```

Comment:
This does unfortunately mean we're importing BS4 js in two spots now. I'm relying on our tests to catch any problems with this. I think sprockets BS4 is still used in some js.erb files floating around, so removing it is out of scope here.